lib/api/controllers/routerController.js

```diff
@@ -418,15 +215,44 @@ module.exports = function RouterController (kuzzle) {
     api.get('/', (request, response) => {
       var _routes = {};
 
-      extractRouteInfo(_routes, routes);
-      extractRouteInfo(_routes, kuzzle.pluginsManager.routes, '/_plugin');
+      extractRouteInfo(_routes, routes, '/' + kuzzle.config.apiVersion);
+      extractRouteInfo(_routes, [{verb: 'get', url: '/_serverInfo', controller: 'read', action: 'serverInfo'}]);
+      extractRouteInfo(_routes, kuzzle.pluginsManager.routes, '/' + kuzzle.config.apiVersion + '/_plugin');
 
       response.setHeader('Access-Control-Allow-Origin', '*');
       response.setHeader('Access-Control-Allow-Headers', 'X-Requested-With');
       response.writeHead(200, {'Content-Type': 'application/json'});
       response.end(stringify({status: 200, error: null, result: {message: 'Available routes for this API version by verb.', routes: _routes}}));
     });
 
+    apiBase.get('/swagger.json', (request, response) => {
+      var content;
+      try {
+        content = fs.readFileSync(__dirname + '/../../../kuzzle-swagger.json');
+      }
+      catch (e) {
+        content = stringify({error: 'The swagger file has not been generated. Use the `kuzzle swagger` CLI command to generate it.', status: 500});
```

Comment:
since the routes are taken directly from the Kuzzle object, the plugins routes are automagically included :)